mmpose/models/losses/multi_task_loss.py

```diff
@@ -0,0 +1,52 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import torch.nn as nn
+
+from mmpose.registry import MODELS
+
+
+@MODELS.register_module()
+class MultiTaskLoss(nn.Module):
+    """Combine multi-task losses together with different factors.
+
+    Args:
+        loss_cfg_list (list): List of Loss Config
+        factors (list): List of Loss factors
+    """
+
+    def __init__(self, loss_cfg_list: list, factors: list):
```

Comment:
About the argument names, I think 'losses' and 'loss_weights' would be simpler and clearer.